scripts/iso-manager.sh

```diff
@@ -157,21 +113,15 @@ _add_archives() {
     # local: ["/srv/scality/metalk8s-2.0.0/", "/tmp/metalk8s-2.1.0.iso"]
     # parsed archives:
     # ("/srv/scality/metalk8s-2.0.0/" "/tmp/metalk8s-2.1.0.iso")
-    IFS=" " read -r -a \
-        archives <<< "$(salt-call pillar.get metalk8s:archives \
-        --out txt | cut -d' ' -f2- | tr -d '[],')"
+    local -a archives=(${CONFIGURED_ARCHIVES[@]+"${CONFIGURED_ARCHIVES[@]}"})
```

Comment:
I'm not sure I understand what it does exactly
```
${CONFIGURED_ARCHIVES[@]+"${CONFIGURED_ARCHIVES[@]}"}
```

Reply:
Same as:
```
local -a archives
if (( ${#CONFIGURED_ARCHIVES[@]} )); then
    archives=("${CONFIGURED_ARCHIVES[@]}")
fi
```

Reply:
Ok, as you want now that I know this, I'm ok to keep this one :smiley:

Reply:
Anyway, this line no longer exist :'( it's in Salt module now.